packages/git-proxy-cli/test/testCli.test.js

```diff
@@ -483,6 +483,114 @@ describe('test git-proxy-cli', function () {
     });
   });
 
+  // *** create user ***
+
+  describe('test git-proxy-cli :: create-user', function () {
```

Comment:
These functions fix one of the failing CLI tests:

```suggestion
  describe('test git-proxy-cli :: create-user', function () {
    before(async function () {
      await helper.addRepoToDb(TEST_REPO_CONFIG);
      await helper.addUserToDb('testuser', 'testpassword', 'test@email.com', 'testGitAccount1');
    });

    after(async function () {
      await helper.removeRepoFromDb(TEST_REPO_CONFIG.name);
      await helper.removeUserFromDb('newuser');
      await helper.removeUserFromDb('testuser');
    });
```